jaxopt/_src/loop.py

```diff
@@ -80,3 +80,11 @@ def while_loop(cond_fun, body_fun, init_val, maxiter, unroll=False, jit=False):
     fun = jax.jit(fun, static_argnums=(0, 1, 3))
 
   return fun(cond_fun, body_fun, init_val, maxiter)
+
+
+def ifelse_cond(jit, cond, if_fun, else_fun, *operands):
```

Comment:
```suggestion
def cond(pred, true_fun, false_fun, *operands, jit=True):
```
for consistency with `lax.cond`